src/com/jalasoft/search_files/search/search_criteria.py

```diff
@@ -0,0 +1,67 @@
+"""
+This module will contain the main criteria for a specific search
+"""
+from src.com.jalasoft.search_files.utils.logging import LOGGER as LOGGER
+
+class Search_Criteria(object):
+    """
+     This class contains the methods that will be used to search.
+    """
+
+    def __init__(self, criteria=3, path='/', advanced_search=False):
```

Comment:
please add a get advance search I will use it in search class

Reply:
you mean flag,.. advance search is already there,.. I'm adding right now a method to get that flag